NOTICE

```diff
@@ -1,4 +1,4 @@
-Copyright 2023 Input Output Global, Inc.
```

Comment:
As far as I understand copyrights reference a year of original publication and don't need to be updated every year.